airflow/executors/base_executor.py

```diff
@@ -182,10 +182,10 @@ def trigger_tasks(self, open_slots: int) -> None:
         sorted_queue = self.order_queued_tasks_by_priority()
 
         for _ in range(min((open_slots, len(self.queued_tasks)))):
-            key, (command, _, _, ti) = sorted_queue.pop(0)
+            key, (command, _, queue, ti) = sorted_queue.pop(0)
             self.queued_tasks.pop(key)
             self.running.add(key)
-            self.execute_async(key=key, command=command, queue=None, executor_config=ti.executor_config)
+            self.execute_async(key=key, command=command, queue=queue, executor_config=ti.executor_config)
```

Comment:
Celery Executor already overrides this method:

https://github.com/apache/airflow/blob/2a2adb3f94cc165014d746102e12f9620f271391/airflow/executors/celery_executor.py#L244-L263

Reply:
Yep celery is not broken as it does override the `trigger_tasks` to pass the queue.

The BaseExecutor should expose all the functionality (like it used to) though without having to have executors override this function? Otherwise the queue argument to `execute_async` is redundant?

Reply:
Oh yeah 👍